app/models/product.rb

```diff
@@ -0,0 +1,15 @@
+class Product < ApplicationRecord
+  
+  has_one_attached :image
+  
+  enum category: {
+    NO_CATEGORY: 0,
+    ENTREE: 1,
+    SECOND_COURSE: 2,
+    DESSERT: 3
```

Comment:
it's not very common in rails enums to use upercase characters, it's usually lowe_case